src/utils/minBigNumber.ts

```diff
@@ -0,0 +1,5 @@
+import { BigNumber } from "@ethersproject/bignumber"
+
+export function minBigNumber(a: BigNumber, b: BigNumber): BigNumber {
+  return a.lt(b) ? b : a
```

Comment:
I think this logic is backwards. should be `min = a.lt(b) ? a : b`. Would be good to write a quick test for this